lib/eu_central_bank.rb

```diff
@@ -136,6 +147,13 @@ def import_rates(format, s, opts = {})
     self
   end
 
+  def check_currency_available(currency)
+    currency_string = currency.to_s
+    return true if currency_string == "EUR"
+    return true if CURRENCIES.include?(currency_string)
+    raise CurrencyUnavailable.new(currency_string)
```

Comment:
Got it, I based that code on a proposal I saw the other day. I'm not very used to use exceptions in Ruby :)